lib/matplotlib/tests/test_datetime.py

```diff
@@ -735,11 +735,40 @@ def test_tricontourf(self):
         fig, ax = plt.subplots()
         ax.tricontourf(...)
 
-    @pytest.mark.xfail(reason="Test for tripcolor not written yet")
     @mpl.style.context("default")
     def test_tripcolor(self):
-        fig, ax = plt.subplots()
-        ax.tripcolor(...)
+        mpl.rcParams["date.converter"] = 'concise'
+        plt.style.use('_mpl-gallery-nogrid')
```

Comment:
```suggestion
```
The gallery-style is not used for tests (and is already set by `@mpl.style.context("default")`).